pkg/metricsstore/metricsstore.go

```diff
@@ -213,6 +221,20 @@ func init() {
 		},
 		[]string{"err_code"},
 	)
+
+	defaultMetricsStore.HTTPResponseTotal = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: metricsRootNamespace,
+		Name:      "http_response_total",
+		Help:      "Counter of HTTP responses sent",
+	}, []string{"code", "method", "path"})
+
+	defaultMetricsStore.HTTPResponseDuration = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: metricsRootNamespace,
+		Name:      "http_response_duration",
+		Buckets:   []float64{.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10},
+		Help:      "Duration in seconds of HTTP responses sent",
```

Comment:
nit: include unit of measurement in description (and maybe name, ie: http_response_duration_ms

Reply:
@steeling Where exactly do you mean by "description" if it's already in the `Help` here?

Reply:
wow i skimmed that poorly... nevermind :)